### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Remove unused md files
- Add a clearer example of the usage of jujulib: listing the details of all the models which covers a lot of things in this package

### CODE REVIEW

File: generator/templates/readme.ts [hunk 1]
```diff
@@ -26,26 +26,63 @@ To access the Juju API, a connection must be made to either a Juju
 controller or a Juju model.
 
 \`\`\`javascript
-import { connect } from "@canonical/jujulib";
+import { connect } from "./dist/api/client";
```
Comment (huwshimi) [inline generator/templates/readme.ts:29]: Should this import be from `@canonical/jujulib` as that's how people will consume it?

File: generator/templates/readme.ts [hunk 2]
```diff
@@ -26,26 +26,63 @@ To access the Juju API, a connection must be made to either a Juju
...
+  );
+  conn = await model.login(credentials);
+
+  // Get the details of the model
+  const client = conn.facades.client;
+  console.log("model details:", await client.fullStatus());
+
+  // Close the connection to the model
+  conn.transport.close();
+}
```
Comment (huwshimi) [inline generator/templates/readme.ts:78]: This is a great example.

**Review by huwshimi — APPROVED** [review]

File: generator/templates/readme.ts (hunk 1 above)
Comment (goulinkh) [reply generator/templates/readme.ts:29]: Nice catch!

File: generator/templates/readme.ts (hunk 2 above)
Comment (goulinkh) [reply generator/templates/readme.ts:78]: thank you!

**Pull request merged**